src/ui/components/screensaver.c

```diff
@@ -30,20 +30,36 @@ static void _render(component_t* component)
     static uint16_t counter = 0;
     counter++;
 
-    const uint16_t slowdown_factor = 6; // slow it down a bunch
+    // these flip between positive and negative when boncing, can also be used to move multiple
+    // pixels per frame
+    static int8_t x_direction = 1;
+    static int8_t y_direction = 1;
+    // setting relative speed for both axes
+    static int8_t x_slowdown = 1;
```

Comment:
those can be `const` instead of `static`.

Reply:
well the slowdowns yes but the directions would probably not really work well as const as they are flipped all the time